cscore/src/main/java/edu/wpi/first/cscore/CvSink.java

```diff
@@ -71,9 +71,23 @@ public CvSink(String name) {
    * @return Frame time, or 0 on error (call GetError() to obtain the error message)
    */
   public long grabFrame(Mat image) {
-    return grabFrame(image, 0.225);
+    return grabFrame(image, 0.225, 0);
   }
 
+  /**
+   * Wait for the next frame and get the image. Times out (returning 0) after 0.225 seconds. The
+   * provided image will have the pixelFormat this class was constructed with.
+   *
```

Comment:
This needs more documentation on the behavior of the function with lastFrameTime being zero or non-zero.  My understanding is that with a non-zero lastFrameTime, this may not wait at all, but just return the most recent frame after that time?

Reply:
Here's my updated wording, it's a little long:

If lastFrameTime is provided and non-zero, the sink will fill image with the first frame from the source that is not equal to lastFrameTime. If lastFrameTime is zero, the time of the current frame owned by the CvSource is used, and this function will block until the connected CvSource provides a new frame.